test/consts/feature_gates.go

```diff
@@ -5,5 +5,5 @@ const (
 	// provided if none are provided by the user. This generally includes features
 	// that are innocuous, or otherwise don't actually get triggered unless the
 	// user takes further action.
-	DefaultFeatureGates = "GatewayAlpha=true,KongServiceFacade=true,KongCustomEntity=true"
+	DefaultFeatureGates = "GatewayAlpha=true,KongServiceFacade=true,RewriteURIs=true,FallbackConfiguration=true,KongCustomEntity=true"
```

Comment:
There exist some assumptions about this value because we're already running tests for different set of feature gates: https://github.com/Kong/kubernetes-ingress-controller/blob/9b863786b5abc262987f3129b046a878511daec7/.github/workflows/_integration_tests.yaml#L77-L143.

Updating this to enable more feature gates would break those assumptions: https://github.com/Kong/kubernetes-ingress-controller/actions/runs/9450920845/job/26030924587?pr=6178

Hence leaving this as is